CMakeLists.txt

```diff
@@ -276,6 +276,11 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
   SET(CMAKE_CXX_ARCHIVE_FINISH "<CMAKE_RANLIB> -no_warning_for_no_symbols -c <TARGET>")
 endif()
 
+if(CPPTRACE_DELAYLOAD_DBGHELP AND BUILD_SHARED_LIBS AND MSVC)
```

Comment:
I think this should also check if we're actually using dbghelp, i.e. `CPPTRACE_UNWIND_WITH_DBGHELP OR CPPTRACE_GET_SYMBOLS_WITH_DBGHELP OR CPPTRACE_DEMANGLE_WITH_WINAPI`. Unfortunately it'll be a bit of a nasty condition :)

Reply:
Please do, thanks!